File: WebApi/Models/DistrictDto.ts
export interface DistrictGetDto {
  id: number;
  name: string;
  erpCode?: string;
  cityId: number;
  cityName?: string;
  createdDate: string;
  updatedDate?: string;
  createdByFullUser?: string;
  updatedByFullUser?: string;
  deletedByFullUser?: string;
}

export interface DistrictCreateDto {
  name: string;
  erpCode?: string;
  cityId: number;
}

export interface DistrictUpdateDto {
  name: string;
  erpCode?: string;
  cityId: number;
}

File: DTOs/CustomerDto/DistrictDto.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace cms_webapi.DTOs
{
    public class DistrictGetDto
    {
        public long Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string? ERPCode { get; set; }
        public long CityId { get; set; }
        public string? CityName { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
        
        // Full user information properties
        public string? CreatedByFullUser { get; set; }
        public string? UpdatedByFullUser { get; set; }
        public string? DeletedByFullUser { get; set; }
    }

    public class DistrictCreateDto
    {
        [Required]
        [MaxLength(100)]
        public string Name { get; set; } = string.Empty;

        [MaxLength(10)]
        public string? ERPCode { get; set; }

        [Required]
        public long CityId { get; set; }
    }

    public class DistrictUpdateDto
    {
        [Required]
        [MaxLength(100)]
        public string Name { get; set; } = string.Empty;

        [MaxLength(10)]
        public string? ERPCode { get; set; }

        [Required]
        public long CityId { get; set; }
    }
}
